File: NearbyPlaces/CustomNavigationBar.m
//
//  CustomNavigationBar.m
//  NearbyPlaces
//
//  Created by Adrian Kubała on 15.02.2017.
//  Copyright © 2017 Adrian Kubała. All rights reserved.
//

#import "CustomNavigationBar.h"
#import "CustomView.h"

@implementation CustomNavigationBar

- (void)drawRect:(CGRect)rect {
  [self drawNavigationBar];
}

- (void)drawNavigationBar {
  CustomView *customView = [[CustomView alloc] initWithView:self];
  [customView drawViewByBezierPathWithOffset:5.0 color:self.barTintColor];
}

@end

File: NearbyPlaces/CustomNavigationBar.swift
//
//  CustomNavigationBar.swift
//  Places
//
//  Created by Adrian on 29.09.2016.
//  Copyright © 2016 Adrian Kubała. All rights reserved.
//

import UIKit

class CustomNavigationBar: UINavigationBar {
  override func draw(_ rect: CGRect) {
    drawNavigationBar()
  }
  
  func drawNavigationBar() {
    let customView = CustomView(view: self)
    customView.drawByBezierPath(withOffset: 5.0, color: barTintColor!)
  }
}
